### ISSUE TITLE
Add another reason to the warning mentioned in section 9.1

### ISSUE THREAD

**Issue by RapidFire05** [tranek/GASDocumentation#60]
9.1 LogAbilitySystem: Warning: Can't activate LocalOnly or LocalPredicted ability %s when not local!
You need to initialize the ASC on the client.

You may also get this warning if you have a local only ability that you grant on the server and try to activate on the server.

**Comment by tranek** [comment]
I don't get that warning when trying to activate a LocalOnly ability from the server. Testing with the meteor ability from this sample project.

![image](https://user-images.githubusercontent.com/1534735/132138419-3f6f4ddc-a3cb-43ef-b04d-dd2772bf3b78.png)


![image](https://user-images.githubusercontent.com/1534735/132138444-067963e9-bfe9-4e00-8d26-ebafbaf88af7.png)

**Comment by ameaninglessname** [comment]
Seeing from UAbilitySystemComponent::InternalTryActivateAbility, it is strange that you don't get a warning log.
The only reason for that is "bIsLocal" is true... on server...

**Comment by RapidFire05** [comment]
I'll check your example and double check my code. Maybe I didn't have the bAllowRemote on. Should be able to look at it today.

**Comment by RapidFire05** [comment]
Just checked again with your example and another one of my own and am NOT getting the warning. So must have been something else. But seems to be gone now. I think what was happening was I was running into some race conditions. I was previously granting ability then activating (if passive) in subsequent BP nodes. I recently changed my passives to activate in GA::OnAvatarSet. 

I tried many different permutations of ways to activate and was not able to recreate the warning so I believe it is fixed. Probably do not need to add anything to section 9.1.
